cmd/statusd/main.go

```diff
@@ -62,6 +62,8 @@ var (
 	// Push Notification
 	enablePN     = flag.Bool("shh.notify", false, "Node is capable of sending Push Notifications")
 	firebaseAuth = flag.String("shh.firebaseauth", "", "FCM Authorization Key used for sending Push Notifications")
+
+	syncAndExit = flag.Int("sync-and-exit", -1, "Timeout in minutes for blockchain sync and exit, zero is infinite")
```

Comment:
I guess there was even a better suggestion from @mandrigin to replace `zero is infinite` with `zero means no timeout unless sync is finished`.

Reply:
Sorry, changing.